awx/settings/defaults.py

```diff
@@ -1011,8 +1011,9 @@ def IS_TESTING(argv=None):
             'formatter': 'simple',
         },
         'external_logger': {
-            'class': 'awx.main.utils.handlers.AWXProxyHandler',
+            'class': 'awx.main.utils.handlers.RSysLogHandler',
             'formatter': 'json',
+            'address': '/var/run/rsyslog/rsyslog.sock',
```

Comment:
if this is specific to the tower-native service instance, I'd name it something more specific. (/var/run/awx-rsyslog/rsyslog.sock?)

Reply:
@wenottingham this would sort of break the pattern set by how we did redis: https://github.com/ansible/tower-packaging/blob/devel/setup/roles/redis/tasks/tasks.yml#L3

But if that is preferable, I'm happy to do it.